trunk/NDHMS/Routing/Reservoirs/module_levelpool.F

```diff
@@ -183,6 +185,7 @@ subroutine LEVELPOOL(ln,qi0,qi1,qo1,ql,dt,H,ar,we,maxh,wc,wl,oe,oc,oa)
     real, intent(IN)    :: we      ! bottom of weir elevation
     real, intent(IN)    :: wc      ! weir coeff.
     real, intent(IN)    :: wl      ! weir length (m)
+    real, intent(IN)    :: dl      ! dam length(m)
```

Comment:
Once more, should be consistent with other names of this variable so that the intent and use of it is clear.  So far, three different variables names to describe the SAME thing, DAML, dam_length, dl.  They should all look the same, and let the encapsulating namespace help differentiate their context.